lib/comfortable_mexican_sofa/extensions/is_taggable.rb

```diff
@@ -82,9 +82,8 @@ def unkeyword!(keyword_ids:)
       # Synchorize the taggings of this item with the ones in its mirrors. So all of them
       # are associated to the same tags
       def sync_mirrors_with_taggings
-        if is_mirrored_with_taggings
-          mirrors.each { |mirror| mirror.assign_new_keywords!(new_keyword_ids: keywords.map(&:id)) }
-        end
+        return unless is_mirrored_with_taggings
+        mirrors.each { |mirror| mirror.assign_new_keywords!(new_keyword_ids: keywords.map(&:id)) }
```

Comment:
I don't think this person exists. 

Remove the `attr_accessor` definition and add

```ruby
def mirrored_with_taggings?
  @mirrored_with_taggings
end

def mirrored_with_taggings=(mirrored)
  @mirrored_with_taggings = mirrored
end
```

Or something to that effect. Or have I not drunk enough coffee this morning?